### ISSUE TITLE
Catalog of Perseus fixes

### ISSUE THREAD

**Issue by whoopsedesy** [sasansom/sedes#53]
Perseus texts are now being maintained at https://github.com/PerseusDL/canonical-greekLit ([documentation](https://github.com/PerseusDL/canonical-greekLit/wiki)). These are more recently updated than what is available at https://www.perseus.tufts.edu/hopper/. Notably, the newer texts internally use Unicode rather than Beta Code ([see notes](https://github.com/PerseusDL/tei-conversion-tools/wiki/Greek-Betacode-to-Unicode-Transformations)).

We have made a [number of fixes](https://github.com/sasansom/sedes/commits/master/corpus) to our copies of Perseus files, which we would like to contribute upstream. We need to make a list of all the fixes, find out which are still relevant to the updated texts, and make pull requests upstream.

This list is currently incomplete, just a placeholder so I remember to create this issue. I think a better way to make the catalog is to go through the list of commits rather than looking at issues, because not all fixes had an associated issue.

- [ ] Canonicalize order of diacritics. The new texts are already Unicode, but if they did not take this into account, the Unicode may not be fully normalizable.
  #45
- [ ] Diacritics preceding letters. May have resulted in combining characters not connected to a base character.
   #9
- [ ] `?`
   #11
- [ ] Contradictory breathing marks
   #46
- [ ] Replace `--` with `_`
   https://github.com/sasansom/sedes/commit/ef61834f0e238294959a79313c9b4001fa9422ff

**Comment by whoopsedesy** [comment]
@sasansom, what does this change represent?
https://github.com/sasansom/sedes/commit/2a8d6cd64d0292abd472e0aa5d2dc5dfc1b2d88c#diff-c4e168f0263854513ee388716b85afe7d1dc35e0efe75cf605a6f3d4e179d920L11680-R11680
Were the erroneous original Beta Code `(` and `)` meant to represent actual parentheses? Should they have become `[1` and `]1` (Beta Code for left and right parentheses), as in 1b96581? Or are they really meant to be square brackets here?

**Comment by whoopsedesy** [comment]
https://github.com/sasansom/sedes/commit/11f2ef8819de9c7514b892da14e5f91ee5cf4721#diff-bf22d475c27688943768eb8f700cbb2d8affac22ae5ff11b3363505db1d618cfL129-R129
https://github.com/PerseusDL/canonical-greekLit/blob/25c5fc1c3ed5dbda4c8f49b2567fe50449fb2b02/data/tlg0533/tlg015/tlg0533.tlg015.perseus-grc1.xml#L130
We deleted the Beta Code `(` and `)`; Perseus changed them to Unicode parentheses. Which is correct?

**Comment by sasansom** [comment]
> @sasansom, what does this change represent?

These brackets signal that the editor, W.H.D. Rouse, considers the line inauthentic or suspect. In the original publication (Loeb 1940-42), Rouse parenthesizes the line and includes a note, "line 151 is only a variant of 150, and something is lost to the effect that those who are not killed in battle are buried in some other way, or not at all" [(pp. 302-3](https://ryanfb.github.io/loebolus-data/L354.pdf)). We changed it to brackets because in beta code parentheses represent rough and soft breathing, and because brackets similarly indicate editorial emendations in other editions.

Glad to hear Perseus now uses Unicode parentheses and preserves Rouse's notation.

**Comment by sasansom** [comment]
> Should they have become `[1` and `]1` (Beta Code for left and right parentheses), as in [1b96581](https://github.com/sasansom/sedes/commit/1b96581b1782ec61c46e0b215b8d73fb8f250f35)?

Yes, that'd be better for the Beta Code, but I think we may have tried it and it didn't work (either in this instance or another one).

**Comment by whoopsedesy** [comment]
> Glad to hear Perseus now uses Unicode parentheses and preserves Rouse's notation.

No, not in Dion. 26.151, they don't. There, they've wrongly interpreted the parentheses as both rough/smooth breathing marks and an opening quotation mark.
```
<l n="151">ʽκ̔τεινομένους κατὰ δῆριν ἐτυμβεύοντο κονίᾐ.</l>
```

It's Callim.Hymn 1.43 where they interpreted the Beta Code `(` `)` as real parentheses (https://github.com/sasansom/sedes/issues/53#issuecomment-830353336).

**Comment by whoopsedesy** [comment]
@sasansom What was the reason for the change in W.D. 473?

https://github.com/sasansom/sedes/commit/a20f03af9b071b7d1e1203d37e8444030dc24eb4#diff-b6e17c0b76521fa791331931b1d69373c3499623567c2cdfa0e2ac5454afcd39
```diff
-<lb />w(=de/ ken a)drosu/nh| sta/xues neu/oien e)/raze,
+<lb />w(=de/ ken a(drosu/nh| sta/xues neu/oien e)/raze,
```

The [book scan](https://archive.org/details/hesiodhomerichym00hesiuoft/hesiodhomerichym00hesiuoft/page/38/mode/1up) has rough breathing over the alpha. This is turning up as a difference against Perseus 6 in #57.

**Comment by whoopsedesy** [comment]
@sasansom At Phaen. 431 [you changed](https://github.com/sasansom/sedes/commit/0441279853bf27446df65223f78da61bfbe3118e) ΚΕΝΤΑΥΡΟΥ to Κενταύρου. New Perseus has κενταψποψ, which is an error that I will fix. But additionally:

Is this word referring to a constellation? If so, I will additionally mark it up as `<name type="constellation">Κενταύρου</name>`, to match other examples like `<name type="constellation">θηρίον</name>` at line 442 on the same page.

[<img width="564" height="309" alt="
δείδιθι, μέχρι βορῆος ἀπαστράψαντος ἴδηαι. (430)
εἰ δέ κεν ἑσπερίης μὲν ἁλὸς KENTAYPOY ἀπείη
ὦμος ὅσον προτέρης, ὀλίγη δέ μιν εἰλύοι ἀχλὺς
αὐτόν, ἀτὰρ μετόπισθεν ἐοικότα σήματα τεύχοι
νὺξ ἐπὶ παμφανόωντι Θυτηρίῳ, οὔ σε μάλα χρὴ
ἐς νότον ἀλλʼ εὔροιο περισκοπέειν ἀνέμοιο.  (435)
δήεις δʼ ἄστρον ἐκεῖνο δύω ὑποκείμενον ἄλλοις·
τοῦ γάρ τοι τὰ μὲν ἀνδρὶ ἐοικότα νειόθι κεῖται
σκορπίου, ἱππούραια δʼ ὑπὸ σφίσι Χηλαὶ ἔχουσιν.
αὐτὰρ ὁ δεξιτερὴν αἰεὶ τανύοντι ἔοικεν
ἀντία δινωτοῖο Θυτηρίου, ἐν δέ οἱ ἀπρὶξ (440)
ἄλλο μάλʼ ἐσφήκωται ἐληλάμενον διὰ χειρὸς
θΗΡΙΟΝ· ὣς γάρ μιν πρότεροι ἐπεφημίξαντο.
" src="https://github.com/user-attachments/assets/43aa8340-8f66-433d-98f0-83428349af2d" />](https://archive.org/details/callimachuslycop00calluoft/page/414/mode/1up)

https://github.com/PerseusDL/canonical-greekLit/blob/dcd11def0e0f441a5f41a64832fe8a03629b4982/data/tlg0653/tlg001/tlg0653.tlg001.perseus-grc2.xml#L561-L574

```xml
<l n="430">δείδιθι, μέχρι βορῆος ἀπαστράψαντος ἴδηαι. </l>
<l n="431">εἰ δέ κεν ἑσπερίης μὲν ἁλὸς κενταψποψ ἀπείη</l>
<l n="432">ὦμος ὅσον προτέρης, ὀλίγη δέ μιν εἰλύοι ἀχλὺς</l>
<l n="433">αὐτόν, ἀτὰρ μετόπισθεν ἐοικότα σήματα τεύχοι</l>
<l n="434">νὺξ ἐπὶ παμφανόωντι Θυτηρίῳ, οὔ σε μάλα χρὴ</l>
<l n="435">ἐς νότον ἀλλʼ εὔροιο περισκοπέειν ἀνέμοιο. </l>
<l n="436" rend="align(indent)">δήεις δʼ ἄστρον ἐκεῖνο δύω ὑποκείμενον ἄλλοις·</l>
<l n="437">τοῦ γάρ τοι τὰ μὲν ἀνδρὶ ἐοικότα νειόθι κεῖται</l>
<l n="438">σκορπίου, ἱππούραια δʼ ὑπὸ σφίσι Χηλαὶ ἔχουσιν.</l>
<l n="439">αὐτὰρ ὁ δεξιτερὴν αἰεὶ τανύοντι ἔοικεν</l>
<l n="440">ἀντία δινωτοῖο Θυτηρίου, ἐν δέ οἱ ἀπρὶξ </l>
<l n="441">ἄλλο μάλʼ ἐσφήκωται ἐληλάμενον διὰ χειρὸς</l>
<l n="442"><name type="constellation">θηρίον</name>· ὣς γάρ μιν πρότεροι ἐπεφημίξαντο.</l>
```

**Comment by whoopsedesy** [comment]
@sasansom In 0738f9dec0be54cc022a4f9d068a148658f64d72 you changed the erroneous beta code `du(??)` to `du\'` at Callim.Hymn 6.94. But the [book scan](https://archive.org/details/callimachuslycop00calluoft/page/132/mode/1up) has acute δύʼ, not grave δὺ’. Perseus 6 [has the acute](https://github.com/PerseusDL/canonical-greekLit/blob/a80f3c09d87cf76e0ba1553d92a063552325c0f9/data/tlg0533/tlg020/tlg0533.tlg020.perseus-grc3.xml#L194). Is the grave accent you added an additional correction to the book scan, on top of the beta code error, or is it a typo on your part and should be an acute?

**Comment by whoopsedesy** [comment]
@sasansom need your comment on a couple of Hom.Hymn issues.

---

At 3.165 ([book scan](https://archive.org/details/hesiodhomerichym00hesiuoft/hesiodhomerichym00hesiuoft/page/336/mode/1up)), SEDES has one word Ἀρτέμιδιξύν where Perseus 6 breaks it into two words Ἀρτέμιδι ξύν. Which is correct?

https://github.com/sasansom/sedes/blob/2194ccce62579dfceb0760d61c4c3f7330a24360/corpus/homerichymns.xml#L853

https://github.com/PerseusDL/canonical-greekLit/blob/a80f3c09d87cf76e0ba1553d92a063552325c0f9/data/tlg0013/tlg003/tlg0013.tlg003.perseus-grc2.xml#L268

https://github.com/PerseusDL/canonical-greekLit/commit/1ab7acce085ce96f1ec3a10c8e6fdade1f18634d#diff-1e6001a2baf80eb1db1b3167dd76ee15032e7a2298aeb36cc8df496dde586335L272 is the commit where Perseus split the word.

---

At 3.181 ([book scan](https://archive.org/details/hesiodhomerichym00hesiuoft/hesiodhomerichym00hesiuoft/page/336/mode/1up)), both SEDES and Perseus disagree with the scan.

* [Scan](https://archive.org/details/hesiodhomerichym00hesiuoft/hesiodhomerichym00hesiuoft/page/336/mode/1up) has περικλύστης
* [SEDES](https://github.com/sasansom/sedes/blob/2194ccce62579dfceb0760d61c4c3f7330a24360/corpus/homerichymns.xml#L873) has περικλύστου
* [Perseus](https://github.com/PerseusDL/canonical-greekLit/blob/a80f3c09d87cf76e0ba1553d92a063552325c0f9/data/tlg0013/tlg003/tlg0013.tlg003.perseus-grc2.xml#L288) has περικλύστοιο

SEDES originally had περικλύστοιο as well, but you changed it to περικλύστου in https://github.com/sasansom/sedes/commit/f5d78adc0645f277c356314cb4d1132cc32d47b0. I'm guessing we want to keep it at περικλύστου in any case. What I don't understand is where περικλύστοιο came from in the first place, because it's not in the scan. Should I offer a correction to Perseus to change περικλύστοιο to περικλύστης?
